proposals/lip-0014.md

```diff
@@ -188,27 +228,27 @@ checkHeadersContradicting(blockHeader1,blockHeader2) {
 
 ### Computing Prevotes and Precommits
 
-All blocks with `maxHeightPreviouslyForged<height` imply `prevotes` and `precommits` as described in [8, Definition 4.1]. In this section, we outline how the total number of `prevotes` and `precommits` can be practically computed using the information contained in the block headers. In the function given below, we only store the current total number of `prevotes` and `precommits` for every block, instead of storing for every block which delegate made a `prevote` or `precommit` for that block. We further assume that the following data structures are available:
+All blocks with `maxHeightPreviouslyForged<height` imply `prevotes` and `precommits` as described in [8, Definition 4.1] with `ρ = LSK_BFT_VOTE_RANGE+1` and `τ=2/3`. In this section, we outline how the total number of `prevotes` and `precommits` can be practically computed using the information contained in the block headers. In the function given below, we only store the current total number of `prevotes` and `precommits` for every block, instead of storing for every block which delegate made a `prevote` or `precommit` for that block. We further assume that the following data structures are available:
 
 *   `blockHeaders[i]`: stores the block header of the current chain at height `i`, the properties of every block header are the same as already described above
 *   `prevoteCounts[i]`: stores the current number of `prevotes` for the block in the chain at height `i`
 *   `precommitCounts[i]`: stores the current number of `precommits` for the block in the chain at height `i`
-*   `delegateMinHeightActive`: for a currently active delegate, this is the height of the first block of the round since when that delegate has been continuously active. We use this information to ensure that the delegate cannot cast  `prevotes` and `precommits` for heights when it was not active. Note that it is sufficient to consider the previous 3 rounds because if  `delegateMinHeightActive<newBlockheader.height-302`, then we can simply set `delegateMinHeightActive:=newBlockheader.height-302` as `prevotes` and `precommits` implied by `newBlockheader` are only cast for blocks of height at least `newBlockheader.height-302`.
+*   `delegateMinHeightActive`: for a currently active delegate, this is the height of the first block of the round since when that delegate has been continuously active. We use this information to ensure that the delegate cannot cast  `prevotes` and `precommits` for heights when it was not active. Note that it is sufficient to consider the previous `LSK_BFT_VOTE_RANGE` blocks because if  `delegateMinHeightActive<newBlockheader.height-LSK_BFT_VOTE_RANGE`, then we can simply set `delegateMinHeightActive:=newBlockheader.height-LSK_BFT_VOTE_RANGE` as `prevotes` and `precommits` implied by `newBlockheader` are only cast for blocks of height at least `newBlockheader.height-LSK_BFT_VOTE_RANGE`.
```

Comment:
```suggestion
*   `delegateMinHeightActive`: for a currently active delegate, this is the height of the first block of the round since when that delegate has been continuously active. We use this information to ensure that the delegate cannot cast `prevotes` and `precommits` for heights when it was not active. Note that it is sufficient to consider the previous `LSK_BFT_VOTE_RANGE` blocks because if `delegateMinHeightActive<newBlockheader.height-LSK_BFT_VOTE_RANGE`, then we can simply set `delegateMinHeightActive:=newBlockheader.height-LSK_BFT_VOTE_RANGE` as `prevotes` and `precommits` implied by `newBlockheader` are only cast for blocks of height at least `newBlockheader.height-LSK_BFT_VOTE_RANGE`.
```